### ISSUE TITLE
[BUG] Ponder crashes with blobToBigInt in decode.ts

### ISSUE THREAD

**Issue by crazyrabbitLTC** [0xOlias/ponder#386]
I am trying to debug an error that isn't giving me too much to work with. 

I'm indexing on an event filter, without a contract address so that I can get every contract that emits the event. 

I'm getting this error, but really having a hard time trying to find where it's happening. My usual tricks of `console.log` and using `try/catch` aren't catching where the error is happening. I've tried process of elimination, but not much luck. The handler I am using, will work fine for many transactions, but then crashes. I'm not able to identify what event is causing it. 

Even with Debug level set to `trace` I'm unable to get an idea of what is happening, and the results are not consistent, it fails at different times. 

It doesn't appear to be the handler code, because if comment out the entire body of the handler such that it's just 
`ponder.on("COMPToken:DelegateVotesChanged", async ({ event, context }) => {})` it still crashes. 

The filter looks like this: 

```
 {
      name: "COMPToken",
      startBlock: 9601359,
      network: "mainnet",
      abi: "./abis/COMPToken.json",
      filter: {
        event: parseAbiItem(
          "event DelegateVotesChanged(address indexed delegate, uint256 previousBalance, uint256 newBalance)")
      }
    },
```

It seems to be failing here in ponder.core: 
https://github.com/0xOlias/ponder/blob/1654494a6d56bd22fe9fbe6aa93a663ba302805a/packages/core/src/utils/decode.ts#L13

This is the error: 
```22:49:00.909 TRACE historical Completed block sync task 10296334, cached from 10296316 (logFilter=COMPToken, network=mainnet)
/Users/dennisonbertram/Develop/gov-indexer/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

TypeError: Cannot read properties of null (reading 'at')
    at blobToBigInt (/Users/dennisonbertram/Develop/gov-indexer/node_modules/@ponder/core/dist/utils/decode.js:14:29)
    at /Users/dennisonbertram/Develop/gov-indexer/node_modules/@ponder/core/dist/event-store/sqlite/store.js:465:64
    at Array.map (<anonymous>)
    at SqliteEventStore.getLogEvents (/Users/dennisonbertram/Develop/gov-indexer/node_modules/@ponder/core/dist/event-store/sqlite/store.js:416:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async EventAggregatorService.getEvents (/Users/dennisonbertram/Develop/gov-indexer/node_modules/@ponder/core/dist/event-aggregator/service.js:64:26)
    at async /Users/dennisonbertram/Develop/gov-indexer/node_modules/@ponder/core/dist/user-handlers/service.js:208:34

Node.js v18.18.1
```

These are the last three lines of the logs: 

```{"level":10,"time":1697324186963,"pid":1365,"hostname":"Dennisons-Air","service":"handlers","msg":"Completed handler (event=\"COMPToken:DelegateVotesChanged\", block=10303224)"}
{"level":30,"time":1697324186963,"pid":1365,"hostname":"Dennisons-Air","service":"handlers","msg":"Processed 2 events (up to Jun 20, 2020)"}
{"level":10,"time":1697324187098,"pid":1365,"hostname":"Dennisons-Air","service":"aggregator","msg":"New historical checkpoint at 1592664441 [Jun 20, 2020] (chainId=1)"}
```

Any idea how I can troubleshoot this?

**Comment by 0xOlias** [comment]
Thanks for the thorough report, and sorry you're running into this. It's definitely an internal Ponder bug.

Few questions:
1. Are you running this against Postgres or SQLite?
2. Does it seem like it's consistently crashing at the same event? If you add `console.log(event.log.id)` in your handler(s), the last one printed before the error message is probably the culprit.
3. Unrelated to the bug - out of curiosity, is the `DelegateVotesChanged` emitted by the children of a factory contract? Wonder if factory contract support would be a good fit here (we're very close to shipping it).

**Comment by crazyrabbitLTC** [comment]
Thanks for getting in touch!

It's running locally against SQLite, not Postgres.

It's crashing on the same event, and I think part of the problem is possibly that there are more events out there with a similar signature, (uint256 vs uint) or different number of arguments.

Maybe that might be part of it?

**Comment by crazyrabbitLTC** [comment]
And no, it's not part of a factory contract I don't think.

**Comment by kyscott18** [comment]
@crazyrabbitLTC I started to sync with the same filter you have in order to reproduce the issue.

In the meantime, it would be helpful for debugging if you could figure out the id of the log that is causing Ponder to crash, with `console.log(event.log.id)`.

**Comment by crazyrabbitLTC** [comment]
I'm having trouble reproducing the error myself, I must have changed something, I'm wondering if it happens when I kill the process, and then restart without deleting the `.ponder` folder. 

i have gotten the error again though in a new project: 

https://github.com/crazyrabbitLTC/ponder-testing

Which might help you. As a note, the graphql api also isn't working in this project for some reason.

**Comment by kyscott18** [comment]
Thanks, this will help. Can you share a link to the abi that you are using.

I believe we fixed this bug in the release 0.0.93, but I will test it to make sure.

**Comment by crazyrabbitLTC** [comment]
Yes just updated, I don't see the problem!

**Comment by crazyrabbitLTC** [comment]
Sorry that was premature, I still have the issue.

**Comment by kyscott18** [comment]
Sorry I wasn't clear, meant to say we fixed the graphQL bug in 0.0.93. I was able to recreate the issued with BlobToBigInt so hope to get that resolved shortly as well
